src/render-delaying-services.js

```diff
@@ -39,15 +39,44 @@ import {getServicePromise} from './service';
 const SERVICES = {
   'amp-dynamic-css-classes': '[custom-element=amp-dynamic-css-classes]',
   'variant': 'amp-experiment',
-  'amp-story': 'amp-story[standalone]',
 };
 
+/**
+ * Base class for render delaying services.
+ * This should be extended to ensure the service
+ * is properly handled
+ */
+export class RenderDelayingService {
```

Comment:
Why is this not an interface?

Reply:
Made changes to make this an interface 😄